docs/evals-introduction.md

```diff
@@ -166,12 +171,12 @@ Live testing requires a Kubernetes cluster and will execute `before-test` and `a
 
 1. **Create Baseline**: Run evaluations with a reference model
    ```bash
-   EXPERIMENT_ID=baseline_gpt4o MODEL=gpt-4o pytest -n 10 ./tests/llm/test_*
+   EXPERIMENT_ID=baseline_gpt4o MODEL=gpt-4o poetry run pytest -n 10 ./tests/llm/test_*
```

Comment:
Need to verify that environment variables like EXPERIMENT_ID are still propogated to pytest this way

Reply:
It's still possible to propogate the env to pytest with poetry.